features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DrawableCopier.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.resources
+
+import android.content.res.Resources
+import android.graphics.drawable.Drawable
+import com.datadog.android.lint.InternalApi
+
+/**
+ * Interface of copying drawable to a new one.
+ */
+@InternalApi
+interface DrawableCopier {
+
+    /**
+     * Called to copy the drawable.
+     * @param originalDrawable the original drawable to copy
+     * @param resources resources of the view.
+     *
+     * @return New copied drawable.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏ "@return new copied drawable"